### PR TITLE
Akhilesh latest US2221

### PR DESCRIPTION
**Pull request by AkhileshKhanduri25**
The latest commits for Barrons, FN, WSJ and Mansion Global Authors Name, Preference Center and Article pages.

### CODE REVIEW

File: panda-qa/Resources/PageObjects/BarronsMailboxPage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/BarronsMailboxPage.robot:1]: //input[@id="iProof0"] is not working for me 

@bhoomraogaripriyanka @ArchanaSaini149

File: panda-qa/Resources/PageObjects/BarronsCapabilitiesPage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/BarronsCapabilitiesPage.robot:1]: /h1[text()="Barrons"] is not working for me, and in my opinion the below path should be in the variables section
dom:${JSFollowButtonBarronsPath}= document.querySelector('ufc-follow-author-widget').shadowRoot  #1
    ...  .querySelector('ufc-follow-widget')  #2
    ...  .querySelector('ufc-follow-button').shadowRoot  #3
    ...  .querySelector('button')  #4

cc @bhoomraogaripriyanka @ArchanaSaini149

File: panda-qa/Resources/PageObjects/BarronsMailboxPage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/BarronsMailboxPage.robot:1]: //input[@id="iProof0"] is not working for me 

cc @bhoomraogaripriyanka @ArchanaSaini149

File: panda-qa/Resources/PageObjects/FNPreferenceCenterPage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/FNPreferenceCenterPage.robot:1]: //*[@id="root"]/div/div/div/div[2]/div/div/div/table[1]/tbody/tr/td[1]/div/div/a  is not validating the toobge pop up

@bhoomraogaripriyanka @ArchanaSaini149

File: panda-qa/Resources/PageObjects/MarketWatchArticlePage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/MarketWatchArticlePage.robot:1]: In my opinion, JS paths should be in the variables section

@bhoomraogaripriyanka @ArchanaSaini149

File: panda-qa/Resources/PageObjects/MarketWatchAuthorsPage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/MarketWatchAuthorsPage.robot:1]: In my opinion, JS paths should be in the variables section

@bhoomraogaripriyanka @ArchanaSaini149

File: panda-qa/Resources/PageObjects/MarketWatchPreferenceCenterPage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/MarketWatchPreferenceCenterPage.robot:1]: //*[@id="root"]/div/div/div/div[2]/div/div/div/table[1]/tbody/tr[1]/td[1]/div/div/a is not working for the toggle pop up

@ArchanaSaini149 @bhoomraogaripriyanka

File: panda-qa/Resources/PageObjects/WSJAuthorsPage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/WSJAuthorsPage.robot:1]: In my opinion, JS paths should be in the variables section

@bhoomraogaripriyanka @ArchanaSaini149

File: panda-qa/Resources/PageObjects/WSJPreferenceCenterPage.robot
Comment (isaiasiniguez) [inline panda-qa/Resources/PageObjects/WSJPreferenceCenterPage.robot:1]: //*[@id="root"]/div/div/div/div[3]/div/ul/li[7] is not working for me

cc @bhoomraogaripriyanka @ArchanaSaini149

File: panda-qa/Resources/PageObjects/BarronsArticlePage.robot
Comment (ArchanaSaini149) [inline panda-qa/Resources/PageObjects/BarronsArticlePage.robot:1]: Line 131 is incomplete. Add data or remove "=" at end.

Comment (AkhileshKhanduri25) [reply panda-qa/Resources/PageObjects/BarronsArticlePage.robot:1]: Ok. Noted.

Comment (AkhileshKhanduri25) [reply panda-qa/Resources/PageObjects/BarronsArticlePage.robot:1]: It's fixed as added the correct values for Follow Button Barrons Path in Variable: ${JSFollowButtonBarronsPath}=  document.querySelector('ufc-follow-author-widget').shadowRoot  #1
    ...  .querySelector('ufc-follow-widget')  #2
    ...  .querySelector('ufc-follow-button').shadowRoot  #3
    ...  .querySelector('button')  #4

**Comment by AkhileshKhanduri25** [comment]
The latest commits for Add Barrons Author names, FN Author names, WSJArticle SignIn Modal and Mansion Global Authors Name,WSJ Preference Center, Barrons preference center, MarketWatch Pereference Center, PEN Follow Button element value, Span text values for Barrons mailbox Page and PEN Click Follow Button Element values are updated.

**Review by ArchanaSaini149 — APPROVED** [review]
Approved.

**Pull request merged**